integration/networktest/traffic/traffic.go

```diff
@@ -0,0 +1,42 @@
+package traffic
+
+import (
+	"testing"
+	"time"
+
+	"github.com/obscuronet/go-obscuro/integration/networktest"
+)
+
+type Runner interface {
```

Comment:
this needs comments.

Reply:
shouldn't this be in  "interfaces.go" as well?
It is one of the main abstractions